dev/com.ibm.ws.jaxrs.2.0_fat/test-applications/restmetrics/src/com/ibm/ws/jaxrs/fat/restmetrics/RestMetricsResource.java

```diff
@@ -268,6 +271,26 @@ public String getMultiParamMessage(
         return RestMetricsResource.message;
     }
 
+    /**
+     * Test when ContainerRequestFilter aborts so the Metrics CRF is never called.
+     * This method should never actually execute
+     *
+     * @return fail message (should never happen)
+     */
+    @GET
+    @Path("/abortTest")
+    @Produces(MediaType.TEXT_PLAIN)
+    public String getAbortTest() {
```

Comment:
Returning a string here will return a 200 response.  I'd suggest returning a `Response` type and then actually return a 500 response that includes the failure message. That way it should be more clear that the test failed.

Reply:
agreed/done